trulens_eval/examples/experimental/virtual_example.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.                Select.RecordCalls.some_other_component.do_something: dict(
</pre><p>The second component here may be more confusing then useful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Removed.